daisy/workflow.go

```diff
@@ -794,9 +794,9 @@ func New() *Workflow {
 
 // NewFromFile reads and unmarshals a workflow file.
 // Recursively reads subworkflow steps as well.
-func NewFromFile(file string) (*Workflow, error) {
+func NewFromFile(file string, varMap map[string]string) (*Workflow, error) {
```

Comment:
since this func is exported, this can break clients who use Daisy as a library.

Reply:
Offline discussed with Andrew, it's not a big deal. It's not breaking the core daisy steps semantic, so it won;t break users' workflow or something. The interface change is not a big problem anymore since we introduced go-modules.